torch/csrc/jit/runtime/graph_executor.cpp

```diff
@@ -439,9 +439,9 @@ struct DifferentiableGraphOp {
     if (v.isTensor()) {
       v = IValue(detach(std::move(v).toTensor()));
     } else if (v.isTensorList()) {
-      c10::List<at::Tensor> lst = std::move(v).toTensorList();
+      std::vector<at::Tensor> lst = v.toTensorVector();
       for (size_t i = 0; i < lst.size(); ++i) {
```

Comment:
could you stick no-clang-tidy in here or change it to a range-based to keep clang-tidy happy?